Sources/Networking/HTTPClient/HTTPRequest.swift

```diff
@@ -11,11 +11,30 @@
 //
 //  Created by Nacho Soto on 2/27/22.
 
+import Foundation
+
 /// A request to be made by ``HTTPClient``
 struct HTTPRequest {
 
     let method: Method
     let path: Path
+    /// If present, this will be used by the server to compute a checksum of the response signed with a private key.
+    let signatureUUID: UUID?
```

Comment:
Hmm well, from the backend PR, seems like the backend is expecting a B64 encoded 16 bytes size string. We could also talk with backend team to change it on their end if we have to

Reply:
Ok let me change that, it's easy.

Reply:
Done